### PR TITLE
Chore/tracefoss 2089 implement role matrix

### PR DESCRIPTION
**Pull request by ds-mmaul**

### CODE REVIEW

**Comment by ds-mmaul** [comment]
> ## [Quality Gate Failed](https://sonarcloud.io/dashboard?id=catenax-ng_tx-traceability-foss_frontend&pullRequest=862) **Quality Gate failed for 'TX Traceability FOSS frontend'**
> Failed conditions
> 
> [6.9% Duplication on New Code](https://sonarcloud.io/component_measures?id=catenax-ng_tx-traceability-foss_frontend&pullRequest=862&metric=new_duplicated_lines_density&view=list) (required ≤ 3%)
> 
> [See analysis details on SonarCloud](https://sonarcloud.io/dashboard?id=catenax-ng_tx-traceability-foss_frontend&pullRequest=862)

Sonar highlights code that's not new. would be <3% if not. Added todo to merge the services into one

File: docs/src/docs/arc42/cross-cutting/safety-security.adoc [hunk 1]
```diff
@@ -13,39 +13,7 @@ JWT token should also contain two claims:
...
-|                  | Delete All                     |            | x            |
-|                  | Delete Own                     | (x)        | x            |
-
-| Administration Panel  | Access "BPN EDC config panel"    |          |            | x
-|                       | Access "Registry lookup Panel"   |          |            | x
-
-|===
-
-Legend: x = full access to all resources, (x) = access to the resources he owns
+You can have a look at the rights and roles matrix in the system overview of the administration guide.
```
Comment (ds-lcapellino) [inline docs/src/docs/arc42/cross-cutting/safety-security.adoc:16]: Please add todo to add the url to the right site.

File: tx-backend/src/main/java/org/eclipse/tractusx/traceability/qualitynotification/application/alert/rest/AlertController.java [hunk 2]
```diff
@@ -44,13 +44,7 @@
...
 import org.springframework.security.access.prepost.PreAuthorize;
 import org.springframework.validation.annotation.Validated;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.ResponseStatus;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```
Comment (ds-lcapellino) [inline tx-backend/src/main/java/org/eclipse/tractusx/traceability/qualitynotification/application/alert/rest/AlertController.java:47]: Please don't squash imports -> can be adjusted in Intellij

File: tx-backend/src/main/java/org/eclipse/tractusx/traceability/qualitynotification/application/investigation/rest/InvestigationsController.java [hunk 3]
```diff
@@ -46,13 +46,7 @@
...
 import org.springframework.security.access.prepost.PreAuthorize;
 import org.springframework.validation.annotation.Validated;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.ResponseStatus;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```
Comment (ds-lcapellino) [inline tx-backend/src/main/java/org/eclipse/tractusx/traceability/qualitynotification/application/investigation/rest/InvestigationsController.java:49]: same as above

File: tx-backend/src/test/java/org/eclipse/tractusx/traceability/integration/assets/DashboardControllerIT.java [hunk 4]
```diff
@@ -44,9 +44,7 @@
 import java.util.stream.Stream;
 
 import static io.restassured.RestAssured.given;
-import static org.eclipse.tractusx.traceability.common.security.JwtRole.ADMIN;
-import static org.eclipse.tractusx.traceability.common.security.JwtRole.SUPERVISOR;
-import static org.eclipse.tractusx.traceability.common.security.JwtRole.USER;
+import static org.eclipse.tractusx.traceability.common.security.JwtRole.*;
```
Comment (ds-lcapellino) [inline tx-backend/src/test/java/org/eclipse/tractusx/traceability/integration/assets/DashboardControllerIT.java:47]: same as above

File: tx-backend/src/test/java/org/eclipse/tractusx/traceability/integration/qualitynotification/alert/PublisherAlertsControllerIT.java [hunk 5]
```diff
@@ -31,11 +31,7 @@
 import org.eclipse.tractusx.traceability.integration.common.support.AlertsSupport;
 import org.eclipse.tractusx.traceability.integration.common.support.AssetsSupport;
 import org.eclipse.tractusx.traceability.qualitynotification.domain.alert.service.AlertsReceiverService;
-import org.eclipse.tractusx.traceability.qualitynotification.domain.base.model.QualityNotificationAffectedPart;
-import org.eclipse.tractusx.traceability.qualitynotification.domain.base.model.QualityNotificationMessage;
-import org.eclipse.tractusx.traceability.qualitynotification.domain.base.model.QualityNotificationSeverity;
-import org.eclipse.tractusx.traceability.qualitynotification.domain.base.model.QualityNotificationStatus;
-import org.eclipse.tractusx.traceability.qualitynotification.domain.base.model.QualityNotificationType;
+import org.eclipse.tractusx.traceability.qualitynotification.domain.base.model.*;
```
Comment (ds-lcapellino) [inline tx-backend/src/test/java/org/eclipse/tractusx/traceability/integration/qualitynotification/alert/PublisherAlertsControllerIT.java:34]: same as above

File: tx-backend/src/test/java/org/eclipse/tractusx/traceability/integration/qualitynotification/alert/PublisherAlertsControllerIT.java [hunk 6]
```diff
@@ -46,18 +42,14 @@
 import org.springframework.transaction.annotation.Transactional;
 import org.testcontainers.shaded.com.fasterxml.jackson.core.JsonProcessingException;
 import org.testcontainers.shaded.com.fasterxml.jackson.databind.ObjectMapper;
-import qualitynotification.base.request.CloseQualityNotificationRequest;
-import qualitynotification.base.request.QualityNotificationSeverityRequest;
-import qualitynotification.base.request.StartQualityNotificationRequest;
-import qualitynotification.base.request.UpdateQualityNotificationRequest;
-import qualitynotification.base.request.UpdateQualityNotificationStatusRequest;
+import qualitynotification.base.request.*;
```
Comment (ds-lcapellino) [inline tx-backend/src/test/java/org/eclipse/tractusx/traceability/integration/qualitynotification/alert/PublisherAlertsControllerIT.java:45]: same as above

File: tx-backend/src/test/java/org/eclipse/tractusx/traceability/integration/qualitynotification/investigation/PublisherInvestigationsControllerIT.java [hunk 7]
```diff
@@ -29,13 +29,7 @@
...
 import org.eclipse.tractusx.traceability.integration.common.support.InvestigationNotificationsSupport;
 import org.eclipse.tractusx.traceability.integration.common.support.InvestigationsSupport;
-import org.eclipse.tractusx.traceability.qualitynotification.domain.base.model.QualityNotificationAffectedPart;
-import org.eclipse.tractusx.traceability.qualitynotification.domain.base.model.QualityNotificationMessage;
-import org.eclipse.tractusx.traceability.qualitynotification.domain.base.model.QualityNotificationSeverity;
-import org.eclipse.tractusx.traceability.qualitynotification.domain.base.model.QualityNotificationStatus;
-import org.eclipse.tractusx.traceability.qualitynotification.domain.base.model.QualityNotificationType;
-import org.eclipse.tractusx.traceability.qualitynotification.domain.base.service.EdcNotificationService;
-import org.eclipse.tractusx.traceability.qualitynotification.domain.base.service.EdcNotificationServiceImpl;
+import org.eclipse.tractusx.traceability.qualitynotification.domain.base.model.*;
```
Comment (ds-lcapellino) [inline tx-backend/src/test/java/org/eclipse/tractusx/traceability/integration/qualitynotification/investigation/PublisherInvestigationsControllerIT.java:32]: same as above

File: tx-backend/src/test/java/org/eclipse/tractusx/traceability/integration/qualitynotification/investigation/PublisherInvestigationsControllerIT.java [hunk 8]
```diff
@@ -44,25 +38,17 @@
...
-import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.transaction.annotation.Transactional;
 import org.testcontainers.shaded.com.fasterxml.jackson.core.JsonProcessingException;
 import org.testcontainers.shaded.com.fasterxml.jackson.databind.ObjectMapper;
-import qualitynotification.base.request.CloseQualityNotificationRequest;
-import qualitynotification.base.request.QualityNotificationSeverityRequest;
-import qualitynotification.base.request.StartQualityNotificationRequest;
-import qualitynotification.base.request.UpdateQualityNotificationRequest;
-import qualitynotification.base.request.UpdateQualityNotificationStatusRequest;
+import qualitynotification.base.request.*;
```
Comment (ds-lcapellino) [inline tx-backend/src/test/java/org/eclipse/tractusx/traceability/integration/qualitynotification/investigation/PublisherInvestigationsControllerIT.java:44]: same as above

**Review by ds-lcapellino — APPROVED** [review]
LGTM

**Pull request merged**